aspnet/web-api/overview/data/using-web-api-with-entity-framework/part-1.md

```diff
@@ -16,17 +16,15 @@ by [Mike Wasson](https://github.com/MikeWasson)
 [Download Completed Project](https://github.com/MikeWasson/BookService)
 
 > This tutorial will teach you the basics of creating a web application with an ASP.NET Web API back end. The tutorial uses Entity Framework 6 for the data layer, and Knockout.js for the client-side JavaScript application. The tutorial also shows how to deploy the app to Azure App Service Web Apps.
-> 
+>
 > ## Software versions used in the tutorial
-> 
-> 
+>
 > - Web API 2.1
-> - [Visual Studio 2013 Update 2](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+> - Visual Studio 2013 (download Visual Studio 2017 [here](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017))
```

Comment:
Since the title of that section is "## Software versions used in the tutorial", I decided to leave it as 2013, because I didn't verify the steps in the remainder of the tutorial on VS 2017 (at this point - but if this topic is >13K page views per month then it will be updated in a subsequent PR). The main purpose of this PR is to replace www.visualstudio.com URLs.